recipes/jfalcou-eve/all/conanfile.py

```diff
@@ -0,0 +1,66 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+required_conan_version = ">=1.33.0"
+
+
+class JfalcouEveConan(ConanFile):
+    name = "jfalcou-eve"
+    description = ("Expressive Velocity Engine - reimplementation of the old "
+                   "Boost.SIMD on C++20"
+                   )
+    homepage = "https://github.com/jfalcou/eve"
+    topics = ("c++", "simd")
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    settings = "os", "arch", "compiler", "build_type"
+    generators = "cmake", "cmake_find_package"
+    no_copy_source = True
```

Comment:
Not sure what went wrong before (perhaps you renamed `name` of this recipe, but not the folder?), but these generators are not needed.
```suggestion
    settings = "os", "arch", "compiler", "build_type"

    no_copy_source = True
```